jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/HttpChannel.java

```diff
@@ -561,16 +560,7 @@ public boolean handle()
                         {
                             if (LOG.isDebugEnabled())
                                 LOG.debug("Could not perform ERROR dispatch, aborting", x);
-                            try
-                            {
-                                _response.resetContent();
-                                sendResponseAndComplete();
-                            }
-                            catch (Throwable t)
-                            {
-                                ExceptionUtil.addSuppressedIfNotAssociated(x, t);
-                                throw x;
-                            }
+                            abort(x);
```

Comment:
The logic is now completely different from before... Is that ok?

Reply:
That's what I'm asking you as the reviewer!
The issue was that we have an exception whilst trying to write an error page and we handle it by trying to write an error page.... which resulted in a nasty loop.   I think aborting if an error page cannot be generated is the best thing to do